json-endpoints/jetpack/class.jetpack-json-api-cron-endpoint.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+// GET /sites/%s/cron
+class Jetpack_JSON_API_Cron_Endpoint extends Jetpack_JSON_API_Endpoint {
+	protected $needed_capabilities = 'manage_options';
+
+	protected function validate_call( $_blog_id, $capability, $check_manage_active = true ) {
+		parent::validate_call( $_blog_id, $capability, false );
+	}
+
+	protected function result() {
+		return array(
+			'cron_array' => _get_cron_array(),
+			'current_timestamp' => time()
+		);
+	}
+
+	protected function sanitize_hook( $hook ) {
+		return preg_replace( '/[^A-Za-z0-9-_]/', '', $hook );
```

Comment:
Do we sync any hooks with uppercase characters?

Can we use `sanitize_key()` instead of rolling our own regex? The difference is that `sanitize_key()` converts uppercase characters to lowercase and also allows dashes.